h/static/styles/partials/_base.scss

```diff
@@ -1,7 +1,23 @@
+/**
+ * 1. Necessary to fix IE floating footer bug
```

Comment:
It may not be clear in future what "IE floating footer bug" refers to. People can look at the git blame output, but including a link to the GitHub issue would be helpful. Additionally can you move the comment to just above the `display: flex` line so it doesn't get detached if for some reason additional styles are added to `html { ... }` in future.

I've verified the fix works. Some Googling turned up https://github.com/philipwalton/flexbugs#flexbug-3 which I think is the issue that we ran into here, correct me if I'm wrong.

Reply:
Yep, I'll add a link to the issue. And the issue you found is the one at work here, yes.